src/mixed-line-bar-chart/chart-container.tsx

```diff
@@ -80,6 +80,26 @@ export interface ChartContainerProps<T extends ChartDataTypes> {
   plotContainerRef: React.RefObject<HTMLDivElement>;
 }
 
+interface XAxisProps {
+  axis: 'X';
+  tickCount: number;
+  scale: ChartScale;
+  ticks: ChartDataTypes[];
+  tickFormatter: TickFormatter;
+  title?: string;
+  ariaRoleDescription?: string;
+}
+
+interface YAxisProps {
+  axis: 'Y';
```

Comment:
minor: I think in other places we use lower-case `'x'` and `'y'` to refer to axis.